dependencies.props

```diff
@@ -9,24 +9,24 @@
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
   <PropertyGroup>
-    <CoreFxCurrentRef>ff616742d18be3af7e9da9327d51502b79de859d</CoreFxCurrentRef>
-    <CoreClrCurrentRef>6de42fd82cb7b8db12fe9339e4304f77343edc83</CoreClrCurrentRef>
+    <CoreFxCurrentRef>a84d35ed9b6b6da8749cf90a2c107f4adfc73a2b</CoreFxCurrentRef>
+    <CoreClrCurrentRef>a84d35ed9b6b6da8749cf90a2c107f4adfc73a2b</CoreClrCurrentRef>
     <ExternalCurrentRef>3b8a99621d89ad9877f053ba8af25e0f25dcd9d8</ExternalCurrentRef>
-    <ProjectNTfsCurrentRef>ff616742d18be3af7e9da9327d51502b79de859d</ProjectNTfsCurrentRef>
-    <ProjectNTfsTestILCCurrentRef>ff616742d18be3af7e9da9327d51502b79de859d</ProjectNTfsTestILCCurrentRef>
+    <ProjectNTfsCurrentRef>a84d35ed9b6b6da8749cf90a2c107f4adfc73a2b</ProjectNTfsCurrentRef>
```

Comment:
The subscription triggers only trigger the update--they aren't passed into the auto-upgrade build at all. (All updates need to happen during any trigger so that the commit that's generated is fresh from GitHub's `HEAD` yet contains all updates.) To disable updating you need to get rid of the `RemoteDependencyBuildInfo`.